pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/pubsub/UpdateWorkflowStatusTest.java

```diff
@@ -69,79 +98,168 @@ private static ParticipantData createParticipantData(String participantIdSeed) {
         participantDataMap.put("MEMBER_TYPE", "SELF");
 
         String ddpParticipantId = TestParticipantUtil.genDDPParticipantId("UpdateWorkflowStatusTest_" + participantIdSeed);
-        return TestParticipantUtil.createParticipantData(ddpParticipantId, participantDataMap, fieldTypeId,
+        return TestParticipantUtil.createParticipantData(ddpParticipantId, participantDataMap, memberTypeFieldTypeId,
                 ddpInstanceDto.getDdpInstanceId(), userDto.getEmailOrThrow());
     }
 
-    private static void deleteParticipantData(int participantDataId) {
-        if (participantDataId >= 0) {
-            participantDataDao.delete(participantDataId);
-        }
-    }
-
-    private static void createFieldSettings() {
+    private static void createMemberTypeFieldSetting() {
         FieldSettingsDto fieldSettingsDto = new FieldSettingsDto.Builder(ddpInstanceDto.getDdpInstanceId())
-                .withFieldType(fieldTypeId)
+                .withFieldType(memberTypeFieldTypeId)
```

Comment:
Given the recent discovery with osteo name differences across environments, are we sure we've got consistently named field types and column names across environments?

Reply:
I'm not sure we do, since DSM users can change or delete these whenever. That is a huge flaw in this "feature". 
Like the many flaws in DSM, we can find a way to bandaid this, or re-implement correctly. 
I don't think a bandaid is appropriate in this PR. What do you think?

Reply:
As long as we know at the moment that the types and column names we're looking for here are consistent, I think we don't need a change in this PR.

Reply:
Actually, I take that back. I do not think these particular field settings are editable by users, but I have to relearn which are and which are not, and where we apply those restrictions. 
Also, *some* field settings are in Liquibase, but not all. In short, it is a mess, but I forget the details of the mess at this point.